### ISSUE TITLE
Results

### ISSUE THREAD

**Issue by asanakoy** [KevinMusgrave/pytorch-metric-learning#22]
Hi,
Thanks for a nice repo!

Do you have some baseline results which you got using you repository for different loss functions?

**Comment by KevinMusgrave** [comment]
@asanakoy Thanks for bringing this up. Results will be here: https://github.com/KevinMusgrave/powerful-benchmarker

I actually had more results up earlier, but took them down because they needed updating. I'll update the spreadsheets soon, and I'll let you know when I've done that.

**Comment by KevinMusgrave** [comment]
@asanakoy I haven't updated the spreadsheets yet, but the spreadsheets will be synced with what appears in my recent paper: https://arxiv.org/abs/2003.08505 (I'll likely be updating the paper as well.)

**Comment by asanakoy** [comment]
Thanks @KevinMusgrave ! 
I just took a liook - nice paper! I was also thinking about the lack of fair comparisons in DML and agree on the issues raised by you.
BTW, if you update your paper you might also cite our paper https://github.com/CompVis/metric-learning-divide-and-conquer as well.

**Comment by KevinMusgrave** [comment]
@asanakoy For sure, it makes sense to add your paper to the related works section.

**Comment by David711nava** [comment]
Tomuch lagg

**Comment by buubakuba58** [comment]
> Cześć,
> dziękuję za miłe repo!
> 
> Czy masz jakieś podstawowe wyniki, które uzyskałeś za pomocą repozytorium dla różnych funkcji utraty?

**Comment by TasinmazhaberCom** [comment]
Sur Yapı Antalya’da dev kampanya
https://www.tasinmazhaber.com/sur-yapi-antalyada-dev-kampanya/

**Comment by mobifcuk** [comment]
nice repo! Exactly I was looking for ;)

**Comment by jamensah29** [comment]
I am new here and i need assistance learn
